dotcom-rendering/src/web/components/CommercialMetrics.tsx

```diff
@@ -31,14 +31,10 @@ export const CommercialMetrics: React.FC<{
 			window.location.hostname.includes('localhost');
 
 		if (isDev || shouldForceMetrics || userIsInSamplingGroup) {
-			sendCommercialMetrics(pageViewId, browserId, Boolean(isDev), adBlockerInUse);
+			sendCommercialMetrics(pageViewId, browserId, isDev, adBlockerInUse);
+			// TODO: force CWV, too
```

Comment:
Why do we want to force core web vitals here? Might be worth extending this comment to give more context?

Reply:
Whenever we capture commercial metrics, we also capture CWV for that page, as we want to ensure that better commercial performance doesn’t come at the expense of user experience.

It’s outside the scope of this PR to address this now, but I’ll expand on the comment.